File: app/src/main/java/me/kashyap/theverge/UiModule.java
package me.kashyap.theverge;

import dagger.Module;
import dagger.Provides;

/**
 * Created on 3/23/2015.
 */
@Module(includes = VergeModule.class)
public class UiModule {
    @Provides
    public MainViewHandler providesMainViewHandler() {
        return new MainHandler();
    }

}

File: app/src/main/kotlin/me/kashyap/theverge/UiModule.kt
package me.kashyap.theverge

import dagger.Module
import dagger.Provides

/**
 * Created on 3/23/2015.
 */
@Module(includes = arrayOf(VergeModule::class))
class UiModule {
    @Provides
    fun providesMainViewHandler(): MainViewHandler {
        return MainHandler()
    }

}
